src/main/java/org/openlmis/fulfillment/extension/point/DefaultOrderCodeGenerator.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * This program is part of the OpenLMIS logistics management information system platform software.
+ * Copyright © 2017 VillageReach
+ *
+ * This program is free software: you can redistribute it and/or modify it under the terms
+ * of the GNU Affero General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or (at your option) any later version.
+ *  
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. 
+ * See the GNU Affero General Public License for more details. You should have received a copy of
+ * the GNU Affero General Public License along with this program. If not, see
+ * http://www.gnu.org/licenses.  For additional information contact info@OpenLMIS.org. 
+ */
+
+package org.openlmis.fulfillment.extension.point;
+
+import org.openlmis.fulfillment.domain.Order;
+import org.openlmis.fulfillment.domain.OrderNumberConfiguration;
+import org.openlmis.fulfillment.repository.OrderNumberConfigurationRepository;
+import org.openlmis.fulfillment.service.referencedata.ProgramDto;
+import org.openlmis.fulfillment.service.referencedata.ProgramReferenceDataService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+/**
+ * A default implementation of OrderCodeGenerator extension point.
+ * @see OrderCodeGenerator
+ */
+@Component("DefaultOrderCodeGenerator")
+public class DefaultOrderCodeGenerator implements OrderCodeGenerator {
```

Comment:
Lets not use "Default" as a name.  While this may be the default option that ships with the software, we need to be more clear with what this implementation does.